aiida/backends/tests/parsers.py

```diff
@@ -104,22 +105,18 @@ def output_test(pk, testname, skip_uuids_from_inputs=[]):
     try:
         to_export.append(c.out.retrieved.dbnode)
     except AttributeError:
-        raise ValueError("No output retrieved node; without it, we cannot "
-                         "test the parser!")
-    export_tree(to_export, folder=folder,
-                also_parents=False,
-                also_calc_outputs=False)
+        raise ValueError("No output retrieved node; without it, we cannot " "test the parser!")
+    export_tree(to_export, folder=folder, also_parents=False, also_calc_outputs=False)
 
     # Create an empty checks file
-    with open(os.path.join(outfolder, '_aiida_checks.json'), 'w') as f:
-        json.dump({}, f)
+    with io.open(os.path.join(outfolder, '_aiida_checks.json'), 'wb', encoding=None) as fhandle:
```

Comment:
`encoding=None` is the default, therefore no need the specify it explicitly

Reply:
I can change this, but I think it's a style choice. I was aiming for 'explicit is better than implicit'.

Reply:
ok